deps.bzl

```diff
@@ -5265,6 +5265,12 @@ def prysm_deps():
         sum = "h1:sI7k6L95XOKS281NhVKOFCUNIvv9e0w4BF8N3u+tCRo=",
         version = "v1.26.0",
     )
+    go_repository(
```

Comment:
Is it necessary to add in that dependency for the PR ? if its for something small we should choose to simply vendor any specific code

Reply:
Yes, sure! I will extract the needed functions from the library and add it to the cmd folder unless you have something else in mind? Thanks!

Reply:
Yes that would be ok, I am not fully sure what the library does. But if its related to general i/o, we can simply add it here:
https://github.com/prysmaticlabs/prysm/tree/develop/io

Reply:
This library is quite straightforward, actually. To provide more information about it, it's lightweight, around 70kB, and its main purpose is to prettify output for better readability.

After spending some time vendoring it, I'm not entirely sure it's worth the effort, to be honest. Even if I strip away some features, like sorting, which aren't used in this PR, I still have to declare some dependencies tied to the package itself in Bazel (such as github.com/mattn/go-runewidth) if I clone it into the io/ package as suggested.

If vendoring isn't really worth it, we're left with two options: either we don't include it and I try to quickly code something similar, or we keep it as is and hope it gets used more in CLI outputs since it's a pretty useful library, if I may say so :)

Let me know what's best for you.

Reply:
> If vendoring isn't really worth it, we're left with two options: either we don't include it and I try to quickly code something similar,

This will most likely be non-trivial to do from scratch.

After talking internally with @nalepae , its fine to add in this dependency. So feel free to include it.